odxtools/compumethods/compuinversevalue.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: MIT
+from dataclasses import dataclass
+from typing import List, Optional
+from xml.etree import ElementTree
+
+from ..odxlink import OdxDocFragment
+
+
+@dataclass
+class CompuInverseValue:
```

Comment:
This looks just like a CompuConst

Reply:
good point. I fixed this by adding making `CompuInverseValue` alias of `CompuConst`  in the latest revision.